tests/fixtures/client.py

```diff
@@ -8,16 +8,17 @@
 from aiohttp.web_routedef import RouteTableDef
 from virtool.config.cls import Config
 from virtool.mongo.identifier import FakeIdProvider
+from virtool.data.utils import get_data_from_app
 from virtool.utils import hash_key
 
 
 class VirtoolTestClient:
-    def __init__(self, test_client):
+    def __init__(self, test_client, settings):
         self._test_client = test_client
 
         self.server = self._test_client.server
         self.app = self.server.app
-        self.settings = self.app["settings"]
+        self.settings = settings
```

Comment:
Is this necessary anymore?

Reply:
You are right, for some reason I thought I needed to maintain the self.settings feature by passing settings in separately, but it's completely unneeded as anything that needs the settings would just call the get_all command.